cmd/qry-gen/gen.go

```diff
@@ -106,7 +109,13 @@ func loadSql(cfg config) (b *bytes.Buffer, err error) {
 	for _, f := range loaded {
 		b.WriteString(fmt.Sprintf("// %s\n", f.Name))
 
-		for _, i := range f.Items {
+		for idx, i := range f.Items {
+			if cfg.comment && unicode.IsUpper([]rune(i.Name)[0]) {
```

Comment:
I think it's ok to generate comments for both exported and unexported constants here, instead of casting string to rune slice and taking it's zero idx. Or atleast, if you prefer to generate comments only for exported queries, cast to rune only single letter, not the whole word.

Reply:
I think it's ok to generate comments only when necessary as they don't add any relevant information